base_report_info/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Base Report Information",
+    "version": "15.0.1.0.0",
+    "category": "Companies",
+    "author": "Quartile Limited",
+    "website": "https://www.quartile.co",
+    "depends": ["base"],
+    "license": "AGPL-3",
```

Comment:
```suggestion
    "license": "LGPL-3",
```
Let's make it LGPL since we may want to let some private modules depend on this module.